Rakefile.rb

```diff
@@ -61,13 +91,13 @@ def name_from(json)
   end
 
   data, _ = json_from('countries.json') rescue {}
+  commit_metadata = file_to_commit_metadata
```

Comment:
In day to day usage (i.e. not the full build that the bot runs), we tend to run this with the `EP_COUNTRY_REFRESH` env variable set, to only update a single country.{1}

At the minute this isn't making any use of that. I think we want to have `file_to_commit_metadata` take a parameter for what directories to examine, and then pass those into the `git log`:

```ruby
commit_metadata = file_to_commit_metadata(countries.values.flatten)
…
def file_to_commit_metadata(dirs)
  command = ['git', '--no-pager', 'log', '--name-only',  '--format=%h|%at', '--', *dirs]   
```

That makes this run in about half the time on my laptop. 

---
{1} technically it could be more than one country as this is just a simple `include?` check, but the standard approach is something like `EP_COUNTRY_REFRESH=Estonia be rake countries.json`